paddle/fluid/framework/ir/graph_helper.cc

```diff
@@ -203,6 +319,29 @@ size_t GraphNum(const Graph &graph) {
   return graph_count;
 }
 
+void CleanIndividualNodes(Graph *graph) {
```

Comment:
Call CleanUnconnectedNode? When can this happen?

Reply:
The original program will create the VarNodes in the beginning. 
After memory reusing, some nodes of them are not needed and no edge links to them, those nodes need to be removed.